numpy/core/tests/test_indexing.py

```diff
@@ -714,7 +709,8 @@ def setUp(self):
             0,
             # Boolean indices, up to 3-d for some special cases of eating up
             # dimensions, also need to test all False
-            np.array(False),
+            # Fixme
```

Comment:
Good with me, it was very useful when there were not many tests, now its not awfully useful (hopefully). How about just removing this one case and maybe adding a comment that there is likely no point in supporting it indefinitely (could also move to its own file?). So that we can basically remove it whenever it should be too much trouble to tweak it?

Just am not sure if I like removing the test completely without a strong need. Who knows maybe it does test some things otherwise not tested, indexing is quite branched with many special cases.